ext/ffi/jit_trampoline.rs

```diff
@@ -217,7 +217,7 @@ mod tests {
       }\n\n",
     );
     assert_codegen(
-      codegen(vec![NativeType::Pointer, NativeType::Pointer], NativeType::U64),
+      codegen(vec![NativeType::Buffer, NativeType::Buffer], NativeType::U64),
```

Comment:
There are now no tests that exercise a `NativeType::Pointer` argument - could you add some?